tests/bdd/features/volume/topology/pool-topology.feature

```diff
@@ -0,0 +1,75 @@
+Feature: Volume Pool Topology
+  The volume topology feature is for selection of particular pools to schedule a replica on.
+  All the volume topology key and values should have an exact match  or the keys must match with pool labels, although pools
+  can have extra labels, for the selection of that pool. The affinity topology label implies the inclusion label 
+  for pool with both key and value or just keys. The storage class has the labels as
+    poolTopologyAffinity: |
+     A1: B1
+     C E : D1
+    which signifies the volume pool topology inclusion labels as {"A1": "B1", "C E" : "D1"}
+
+    poolHasTopologyKey: |
+      alpha
+      beta    
+   which signifies the volume pool topology inclusion labels as {"alpha": "", "beta" : ""} 
+
+  Background:
+    Given a control plane, three Io-Engine instances, nine pools
+
+
+# The labels to be applied to the pools.
+##########################################################################################
+#   Description            ||   Pool Name   ||         Label      ||        Node        ||
+#=========================================================================================
+#     "pool1" has the      ||   node1pool1  ||     alpha1=beta1   ||     io-engine-1    || 
+#  label "alpha1=beta1"    ||   node2pool1  ||     alpha1=beta1   ||     io-engine-2    ||
+#                          ||   node3pool1  ||     alpha1=beta1   ||     io-engine-3    ||
```

Comment:
would it make more sense to group these by node?, example:
```suggestion
#     "pool1" has the      ||   node1pool1  ||     alpha1=beta1   ||                              || 
#  label "alpha1=beta1"    ||   node1pool2  ||     alpha1=beta1   ||     io-engine-2    ||
#                          ||   node1pool3  ||     alpha1=beta1   ||                              ||
```

Reply:
Here the priority is pool labels . so its grouped by pools . like pool1 on node 1, node2 , node 3 has the labesl `zone-us=us-west-1`